ansible/inventories/dev/hosts

```diff
@@ -0,0 +1,2 @@
+[linux]
```

Comment:
at sometime we'll need to main inventory of our deployment in single file. 
not important rn